tcms/issuetracker/types.py

```diff
@@ -336,23 +336,34 @@ class Redmine(IssueTrackerType):
     Support for Redmine. Requires:
 
     :base_url: the URL for this Redmine instance. For example http://redmine.example.org:3000
-    :api_username: a username registered in Redmine
-    :api_password: the password for this username
+    :api_username: (Optional) A username registered in Redmine. If omitted, the :api_password: field will be treated as an API access token.
+    :api_password: Either the password for the Redmine user (when :api_username: is provided), or the API access token (when :api_username: is empty).
+
+    Users can manage or generate their Redmine API tokens by visiting: https://www.redmine.org/my/account → "API access key"
+    (or <your-redmine-instance>/my/account if self-hosted)
     """
 
     def is_adding_testcase_to_issue_disabled(self):
         (api_username, api_password) = self.rpc_credentials
```

Comment:
`api_username` variable is now unused and triggers various linter errors - see pylint. Please prefix with an underscore to silence these errors and keep code readability.